peer/brontide.go

```diff
@@ -51,24 +51,24 @@ import (
 
 const (
 	// pingInterval is the interval at which ping messages are sent.
-	pingInterval = 1 * time.Minute
+	pingInterval = 3 * time.Minute
```

Comment:
I thought we only wanted to do this if we're in tor mode, or see an inbound connection from a tor peer?

Reply:
We will absolutely do it that way if we confirm that this actually works :). This was a fast way for me to get a test branch out to @Liongrass and @bitromortac to validate that it would fix their experience. If it works I'll reimplement for tor-only. If you want me to do it anyway without the results, I can.

Reply:
I still had many disconnects for almost all peers with the 4k message max. I'm running the latest commit, to see if things change.

Reply:
ping timeout disconnections are (almost) absent with the 90s, I'm trying with reduced parameters

Reply:
I took some measurements for round trip times:

Median: 1.5 s
95th percentile: 5.3 s
99th percentile: 13.9 s
99.5th percentile: 22.9 s
99.9th percentile: 54.8 s

I think anywhere between 20-90s for the timeout should be ok